Resources/config/discovery.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="httplug.client.default"
+             class="Http\Client\HttpClient">
```

Comment:
depending on the decision on https://github.com/php-http/httplug/issues/68#issuecomment-149112039 this might need to be changed